ibm/service/vpc/resource_ibm_is_instance.go

```diff
@@ -158,6 +160,13 @@ func ResourceIBMISInstance() *schema.Resource {
 		),
 
 		Schema: map[string]*schema.Schema{
+			isInstanceAvailablePolicyHostFailure: {
+				Type:        schema.TypeString,
```

Comment:
Will it be both optional and computed..

When user provisions a VM without any availability_policy_host_failure what is the value set in computed

Reply:
it is set with teh default value from the API, which is "restart"